style.css

```diff
@@ -79,3 +81,187 @@ header a {
   margin-right: 24px;
   color: #6070ff;
 }
+
+.project {
+  margin: 114px 24px 114px 24px;
+  display: grid;
+  gap: 88px;
+}
+
+.project-card {
+  box-sizing: border-box;
+  border-radius: 16px;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+}
```

Comment:
- [ ] For proper alignment of the child elements in the `project-card`, use `flex-box` which is a good practice way and a learing objective of this project.